### ISSUE TITLE
Re python resources

### ISSUE THREAD

**Issue by Ecofoot** [BostonPython/about#41]
Add section of popular online free sites to learn python aspects
     -rate sites by popularity eventually

**Comment by jonkiparsky** [comment]
@Ecofoot Do you have specific sites that you'd like to add?

**Comment by Ecofoot** [comment]
> @Ecofoot Do you have specific sites that you'd like to add?

No, not at them moment but I will send a list of sites that I've found helpful in the past when I get a chance. 

The group could vote/rank any sites given too which would be good to do.

A listing of e-news sites focusing on data science might be good too.  I follow KDnuggets, Analytic Vidhya, ODSC, Medium newsletters as main ones.
